components/recorder-cli/mocha/index.mjs

```diff
@@ -27,7 +30,22 @@ import {
   addSocketListener,
 } from "../../socket/index.mjs";
 
-const { undefined, parseInt, Promise } = globalThis;
+const { undefined, parseInt, Boolean, Promise } = globalThis;
+
+function buildTestFailure(error) {
```

Comment:
The rest of the code prefers arrows. It makes it clear that it is not a method.

Reply:
I actually prefer `function` which makes it visually apparent that it's function, plus hoisting allows you to move utility functions to the end of the file or block. But ok, I'll change it to keep with the existing style.